retroarch/retroarch.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "retroarch",
+	"config-opts": [ "--enable-dbus" ],
+	"make-args": [ "GLOBAL_CONFIG_DIR=${FLATPAK_DEST}/etc" ],
+	"sources": [
+		{
+			"type": "git",
+			"url": "https://github.com/libretro/RetroArch.git",
+			"branch": "9b282aa742b6c3d2f2925ae5a12e2cd7c6b6ad38"
+		},
+		{
+			"type": "file",
+			"path": "org.libretro.RetroArch/retroarch.cfg"
+		}
+	],
+	"post-install": [
+		"mkdir -p ${FLATPAK_DEST}/etc",
+		"sed s:@prefix@:${FLATPAK_DEST}:g retroarch.cfg > ${FLATPAK_DEST}/etc/retroarch.cfg"
+	],
+	"modules": [
+		"org.libretro.RetroArch/libpng/libpng-1.6.35.json",
+		"org.libretro.RetroArch/nvidia-cg-toolkit/nvidia-cg-toolkit-3.1.0013.json",
+		"org.libretro.RetroArch/shared-modules/SDL/SDL-1.2.15.json",
+		"org.libretro.RetroArch/shared-modules/SDL/SDL_image-1.2.12.json",
+		"org.libretro.RetroArch/shared-modules/SDL/SDL_mixer-1.2.12.json",
+		"org.libretro.RetroArch/shared-modules/SDL/SDL_net-1.2.8.json",
+		"org.libretro.RetroArch/shared-modules/SDL/SDL_ttf-2.0.11.json",
+		"org.libretro.RetroArch/shared-modules/libusb/libusb.json",
+		"org.libretro.RetroArch/shared-modules/gudev/gudev.json",
+		"org.libretro.RetroArch/libbz2/libbz2-1.0.8.json",
+		"org.libretro.RetroArch/xrandr/xrandr-1.5.1.json",
+		"org.libretro.RetroArch/libaio/libaio-0.3.112.json",
+		"org.libretro.RetroArch/shared-modules/glu/glu-9.json",
```

Comment:
you should use the shared module

Reply:
It is. These are all from the [RetroArch flatpak](https://github.com/flathub/org.libretro.RetroArch/blob/master/org.libretro.RetroArch.json#L57)

Reply:
It looks like the initial PR for adding RetroArch didn't have any of these. I wonder if they are actually required. They might be needed for bugfixes, but I'm not sure. I'll see if I can build without them and test it to make sure it works as expected.

Reply:
Okay, it looks like the flatpak build wasn't actually even building any of these, so they clearly aren't required. I don't know why the RetroArch maintainers added them. Everything seems to be working fine without them, so I will remove them.